src/test/kotlin/net/maizegenetics/phgv2/pathing/MapKmersTest.kt

```diff
@@ -252,26 +253,46 @@ class MapKmersTest {
 
         val refRangeIdToHapIdMap = graph.refRangeIdToHapIdMap()
 
-        //loop through the kmers in kmerHashToLong Map
-        loadedKmerMapData.kmerHashToLongMap.long2LongEntrySet().forEach { (kmerHash,encodedOffset) ->
+
+        loadedKmerMapData.kmerHashToLongMap.long2ObjectEntrySet().forEach {(kmerHash, values) ->
             val kmerHashLong = kmerHash.toLong()
             assertTrue(kmerMapToHapids.containsKey(kmerHashLong))
 
-            val (refRangeId, offset) = AlignmentUtils.decodeRangeIdAndOffset(encodedOffset)
-            //get the ref range so we can look up refRange -> HapId Index
-            val hapIdToIndexMap = refRangeIdToHapIdMap[refRangeId]!!
-
+            //Loop through the values and test the kmers keeping track of the hapIds which are not in any known refRange.
             val hapIds = kmerMapToHapids[kmerHashLong]!!
+            val unSeenHapIds = mutableSetOf<String>()
+            unSeenHapIds.addAll(hapIds)
+
+            println("Kmer: $kmerHashLong")
```

Comment:
Might want to remove (or leave for debugging but comment out) this line if it going to be printed alot.  Could be distracting in the log file